domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/ScannedDocumentSubtype.java

```diff
@@ -0,0 +1,12 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+public enum ScannedDocumentSubtype {
+
+    OCON9X("OCON9x");
```

Comment:
There are more subtypes than this. I would probably add a comment that this is not an exhaustive list of subtypes to avoid confusion.

Reply:
Fixed in next commit